dlc_developer_config.toml

```diff
@@ -46,14 +46,14 @@ ec2_tests = true
 
 ### SM specific tests
 ### Off by default
-sagemaker_local_tests = false
```

Comment:
Normally for PR builds we don't require sm tests. Not sure if we need these in this case